spec/javascripts/health-care/health-care.spec.js

```diff
@@ -201,5 +211,31 @@ describe("Health Care Form", function() {
         expect(true).to.be.false;
       }).always(function() { done(); });
   });
+
+  it("should return true for valid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var validInput = ['55000.00', '100', '1.99'];
+
+    for (var i = 0; i < validInput.length; i++) {
+      testElement.value = validInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.true;
+    }
+  });
+
+  it("should return false for invalid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var invalidInput = ['1,000', 'abc', '$100'];
+
+    for (var i = 0; i < invalidInput.length; i++) {
+      testElement.value = invalidInput[i];
+      var valid = HealthApp.validate(testElement);
+      expect(valid).to.be.false;
+    }
+  });
```

Comment:
Do you mean a test that will make sure that the `validateMonetaryValue` function runs on all of the DOM nodes with the `data-validation-type="monetary"`? For that, would we need to make sure the function is called? Not entirely sure how to do this, looks like we can use Sinon spies?